examples/android-gradle/build.gradle

```diff
@@ -1,18 +1,8 @@
-buildscript {
```

Comment:
i think this was here intentional to test the plugin works with the deprecated buildscript setup. But I guess it's not that relevant anymore so we should be good

Reply:
Oh, I can’t foresee a scenario where the new plugin setup would work but this would not work, but happy to add it back if you think it is needed or that happened in the past.

Reply:
nope it's all good 👍